data/i18n/hi/hi.toml

```
@@ -1,180 +1,180 @@
+other = "कुबेरनेट्स विशेषताएं"
 
 [main_cncf_project]
-other = """We are a <a href="https://cncf.io/">CNCF</a> graduated project</p>"""
+other = """हम एक <a href="https://cncf.io/">CNCF</a> स्नातक परियोजना</p> हैं"""
 
 [main_kubeweekly_baseline]
-other = "Interested in receiving the latest Kubernetes news? Sign up for KubeWeekly."
+other = "नवीनतम कुबेरनेट्स समाचार प्राप्त करने के इच्छुक हैं? KubeWeekly के लिए साइन अप करें।"
 
 [main_kubernetes_past_link]
-other = "View past newsletters"
+other = "पिछले न्यूज़लेटर देखें"
 
 [main_kubeweekly_signup]
-other = "Subscribe"
+other = "सब्सक्राइब"
 
 [main_contribute]
-other = "Contribute"
+other = "योगदान"
 
 [main_edit_this_page]
-other = "Edit This Page"
+other = "इस पृष्ठ को संपादित करें"
 
 [main_page_history]
-other ="Page History"
+other ="पृष्ठ इतिहास"
 
 [main_page_last_modified_on]
-other = "Page last modified on"
+other = "पृष्ठ अंतिम बार संशोधित किया गया"
 
 [main_by]
-other = "by"
+other = "द्वारा"
 
 [main_documentation_license]
-other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
+other = """कुबेरनेट्स लेखक | <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a> के अंतर्गत वितरित दस्तावेज़"""
 
 [main_copyright_notice]
-other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
+other = """Linux फाउंडेशन &reg;। सभी अधिकार सुरक्षित। Linux फाउंडेशन ने ट्रेडमार्क पंजीकृत किया है और ट्रेडमार्क का उपयोग करता है। Linux फाउंडेशन के ट्रेडमार्क की सूची के लिए, कृपया हमारा <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">ट्रेडमार्क उपयोग पृष्ठ</a> देखें।"""
 
 # Labels for the docs portal home page.
 [docs_label_browse]
-other = "Browse Docs"
+other = "प्रलेखन ब्राउज़ करें"
 
 [docs_label_contributors]
-other = "Contributors"
+other = "योगदानकर्ता"
 
 [docs_label_users]
-other = "Users"
+other = "उपयोगकर्ता"
 
 [docs_label_i_am]
-other = "I AM..."
+other = "मैं हूँ..."
 
 # layouts > blog > pager
 
 [layouts_blog_pager_prev]
-other = "<< Prev"
+other = "<< पिछला"
 
 [layouts_blog_pager_next]
-other = "Next >>"
+other = "अगला >>"
 
 # layouts > blog > list
 
 [layouts_case_studies_list_tell]
-other = "Tell your story"
+other = "अपनी कहानी बतायें"
 
 # layouts > docs > glossary
 
 [layouts_docs_glossary_description]
-other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to K8s, as well as more general terms that provide useful context."
+other = "इस शब्दावली का उद्देश्य कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची है। इसमें K8s के लिए विशिष्ट तकनीकी शब्द और उपयोगी संदर्भ प्रदान करने वाले सामान्य शब्द शामिल हैं।"
```

I actually recommend either:
```suggestion
other = "इस शब्दावली का उद्देश्य कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची है। इसमें Kubernetes के लिए विशिष्ट तकनीकी शब्द और उपयोगी संदर्भ प्रदान करने वाले सामान्य शब्द शामिल हैं।"
```
or
```suggestion
other = "इस शब्दावली का उद्देश्य कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची है। इसमें कुबेरनेट्स के लिए विशिष्ट तकनीकी शब्द और उपयोगी संदर्भ प्रदान करने वाले सामान्य शब्द शामिल हैं।"
```

---

2nd suggestion is fine I guess.

---

Yes, `k8s` -> `कुबेरनेट्स`